sdl_android/src/main/java/com/smartdevicelink/proxy/interfaces/ISdl.java

```diff
@@ -176,4 +176,10 @@
      */
     SdlMsgVersion getSdlMsgVersion();
 
+    /**
+     * Get wiProVersion
+     * @return byte representing wiProVersion
+     */
+    byte getWiProVersion();
```

Comment:
We should not continue the misleading naming of the protocol layer. This method should be `Version getProtocolVersion`